api/src/API/Controllers/RacesController.cs

```diff
@@ -40,17 +43,78 @@ public async Task<IActionResult> GetAll()
             return Ok(result);
         }
 
+        [AllowAnonymous]
+        [HttpGet("public")]
+        public async Task<IActionResult> GetAllPublic()
+        {
+            RaceContainerClient container = containerProvider.RaceContainer;
+            IEnumerable<Race> result = await container.GetAllAsync();
```

Comment:
this is incorrect; it needs to be `container.GetManyAsync(it => it.Where(race => race.IsPublic))`

Reply:
Good catch. Thanks